src/pages/organisaties.jsx

```diff
@@ -0,0 +1,34 @@
+import { SkipLink } from "@utrecht/component-library-react/dist/css-module";
+import * as React from "react";
+import Footer from "../components/Footer.js";
+import Header from "../components/header.jsx";
+import Content from "../components/organisaties-content.jsx";
+import "./index.css";
+import "@utrecht/design-tokens/dist/index.css";
+import "../fonts/sourcesanspro.ttf";
```

Comment:
dit moet weg de font link op index is al genoeg en ik mis de alignment classname en style link